$NetBSD: patch-jisyo,v 1.1.1.1 2002/05/31 12:59:59 seb Exp $

Dictionary (SKK-JISYO.L) corrigenda.
Partially from skk-dic mailing list.

--- jisyo/SKK-JISYO.L.fix1	Mon Dec 20 16:28:52 1999
+++ jisyo/SKK-JISYO.L	Mon Dec 27 23:53:47 1999
@@ -3530,7 +3530,7 @@
 しりあe /知り合/
 しらべr /調べ/
 しらじらs /白々/
-しらs /報/知ら/知らせ/
+しらs /報/知ら/
 しらm /白/
 しらk /白/
 しらb /調/
@@ -4216,9 +4216,9 @@
 ぐi /食/
 くんよm /訓読/
 くんづk /君付/
-くわわt /加/
+くわわt /加わ/
 くわだt /企/
-くわw /加わ/
+くわw /加/
 くわs /詳/加/
 くわk /区分/
 くわe /加/咥/銜/加え/
@@ -30746,7 +30746,7 @@
 がいかんび /外観美/
 がいき /外気/
 がいきおん /外気温/
-がいきょう /概況/外匡/
+がいきょう /概況/外筐/
 がいきょく /外局/
 がいきれいぼう /外気冷房/
 がいきん /外勤/
@@ -34044,7 +34044,7 @@
 きゅうきゅうたい /救急隊/
 きゅうきゅうばこ /救急箱/
 きゅうきゅうばんそうこう /救急絆創膏/
-きゅうきょ /急拠/急遽/急きょ/
+きゅうきょ /急遽/急きょ/
 きゅうきょう /窮境/
 きゅうきょうさんけんしょこく /旧共産圏諸国/
 きゅうきょうしょくいん /旧教職員/
@@ -35040,9 +35040,9 @@
 きょうぞんきょうえい /共存共栄/
 きょうぞんほう /共存法/
 きょうぞんほうほう /共存方法/
-きょうたい /匡体/筐体/狂態/矯態/嬌態/筺体/
+きょうたい /筐体/狂態/嬌態/
 きょうたいいき /狭帯域/
-きょうたいじっそうぎじゅつ /匡体実装技術/
+きょうたいじっそうぎじゅつ /筐体実装技術/
 きょうたく /供託/教卓/
 きょうたくきん /供託金/
 きょうたくしょ /供託所/
@@ -36904,7 +36904,7 @@
 ぎじんほう /擬人法/
 ぎせい /犠牲/擬声/擬制/
 ぎせいご /擬声語/
-ぎせいしゃ /儀牲者/犠牲者/
+ぎせいしゃ /犠牲者/
 ぎせき /議席/
 ぎせきすう /議席数/
 ぎせきはいぶん /議席配分/
@@ -42867,7 +42867,7 @@
 こうぎせきぶん /広義積分/
 こうぎたいしょう /抗議対象/
 こうぎちゅう /講義中/
-こうぎとう /講義搭/講義棟/
+こうぎとう /講義棟/
 こうぎないよう /講義内容/
 こうぎぶん /抗議文/
 こうぎめい /講義名/
@@ -84011,7 +84011,7 @@
 とうじゅうろう /藤十郎/
 とうじゆうめい /当時有名/
 とうじょ /唐書/
-とうじょう /登場/搭乗/東條/東条/棟上/東上/塔乗/頭状/東城/筒状/
+とうじょう /登場/搭乗/東條/東条/棟上/東上/頭状/東城/筒状/
 とうじょういぜん /登場以前/
 とうじょういらい /登場以来/
 とうじょういん /搭乗員/
@@ -84025,7 +84025,7 @@
 とうじょうご /登場後/
 とうじょうしゃ /搭乗者/
 とうじょうしゃほけん /搭乗者保険/
-とうじょうじ /塔乗時/登場時/
+とうじょうじ /搭乗時/登場時/
 とうじょうじっせき /搭乗実績/
 とうじょうじんぶつ /登場人物/
 とうじょうじんぶつたち /登場人物達/
@@ -84038,7 +84038,7 @@
 とうじょうとうじ /登場当時/
 とうじょうばめん /登場場面/
 とうじょうばんぐみ /登場番組/
-とうじょうび /塔乗日/搭乗日/
+とうじょうび /搭乗日/
 とうじょうびん /搭乗便/
 とうじょうまえ /登場前/搭乗前/
 とうじょうよてい /登場予定/搭乗予定/
@@ -93140,7 +93140,7 @@
 はげたか /禿鷹/
 はげやま /禿山/
 はげん /波源/
-はこ /箱/函/運/葉子/□/匡/凾/筥/箪/匣/
+はこ /箱/函/運/葉子/□/筐/凾/筥/箪/匣/
 はこいり /箱入り/箱入/
 はこう /破口/跛行/波高/
 はこうぶんぷ /波高分布/
